resources/metadata/cloudFoundryCreateService.yaml

```diff
@@ -0,0 +1,157 @@
+metadata:
+  name: cloudFoundryCreateService
+  description: cloudFoundryCreateService
+  longDescription: |
+    Create CloudFoundryService
```

Comment:
Please improve the description and long description

Reply:
Analog to prerequisites restructuring. Long description has been improved.